pants.ini

```diff
@@ -21,14 +21,15 @@ pythonpath: [
   ]
 backend_packages: +[
     'twitter.common.pants.python.commons',
-    'pants.contrib.python.checks',
```

Comment:
intentional?

Reply:
Yes: Putting something in the plugin list enables it, so having this here was enabling it twice, which now causes an error.

Reply:
ah, cool